loadtest/config.go

```diff
@@ -69,12 +70,15 @@ type UserControllerConfiguration struct {
 	//   UserControllerSimple - A simple version of a controller.
 	//   UserControllerSimulative - A more realistic controller.
 	Type userControllerType
-	// A rate multiplier that will affect the speed at which user actions are
+	// A distribution of rate multipliers that will affect the speed at which user actions are
 	// executed by the UserController.
 	// A Rate of < 1.0 will run actions at a faster pace.
 	// A Rate of 1.0 will run actions at the default pace.
 	// A Rate > 1.0 will run actions at a slower pace.
-	Rate float64
+	RatesDistribution []struct {
+		Rate       float64
+		Percentage float64
+	}
```

Comment:
@isacikgoz  I hope this anon struct won't cause too much troubles with https://github.com/mattermost/mattermost-load-test-ng/pull/257. If so, let me know :)